11-deviceservices.md

```diff
@@ -5,31 +5,30 @@ layout: default
 ---
 
 # Device Services (Orca Service)
+
 This is a wrapper for the Orca device services
 
 The first step is to register for device activation to generate an activation code.  This code can then register the device and generate the secret API key to use for other services.
 
-
-## Create an OrcaServiceConfig object for settings.
+> Create an OrcaServiceConfig object for settings.
 
 {% highlight csharp %}
 var config = new HpsOrcaServiceConfig
 {
-	DeviceId = 5315938,
-	LicenseId = 101433,
-	UserName = "777700857994",
-	Password = "$Test1234",
-	SiteId = 101436,
-	SiteTrace = "trace0001",
-	VersionNumber = "1234",
-	IsTest = true, //Use for certifications and develpoment testing.  false by default.
-	ApplicationId = "Mobuyle Retail",
-	HardwareTypeName = "Heartland Mobuyle"
+	  DeviceId = 5315938,
+	  LicenseId = 101433,
+	  UserName = "777700857994",
+	  Password = "$Test1234",
+	  SiteId = 101436,
+	  SiteTrace = "trace0001",
+	  VersionNumber = "1234",
+	  IsTest = true, //Use for certifications and develpoment testing.  false by default.
+	  ApplicationId = "Mobuyle Retail",
+	  HardwareTypeName = "Heartland Mobuyle"
```

Comment:
@ttnnkkrr Yep, we can get that in another PR. This PR was mainly to make some organizational tweaks to make the design work in the dev portal. Content updates will come later